app/screens/edit_post/edit_post.js

```diff
@@ -108,7 +108,7 @@ export default class EditPost extends PureComponent {
     };
 
     focus = () => {
-        this.messageInput.refs.wrappedInstance.focus();
```

Comment:
could not be found and caused the app to crash

Reply:
Should we be changing in all instances?

Reply:
this is the only instance where we were using the ref

Reply:
<img width="685" alt="screenshot 2018-11-26 at 5 33 18 pm" src="https://user-images.githubusercontent.com/4973621/49013029-69b07200-f1a1-11e8-81c0-1c08afea8416.png">
I see few instances where we use wrappedInstance on refs.

Reply:
those are not wrappedInstances created by injectIntl, those are from connected components

Reply:
but you are right with some of them yes!!! let me fix that.. thanks

Reply:
No problem. We can do that in a separate PR if you like

Reply:
@sudheerDev already done